lib/SILGen/Condition.cpp

```diff
@@ -39,7 +39,7 @@ static SILLocation getContinuationLoc(SILBasicBlock &BB, SILLocation Fallback) {
 void Condition::exitTrue(SILGenFunction &SGF, ArrayRef<SILValue> Args) {
   // If there's no continuation block, it's because the condition was
   // folded to true.  In that case, we just continue emitting code as
-  // if we were still in the true case, and we're unreachable iff the
+  // if we were still in the true case, and we're unreachable if and only if the
```

Comment:
Why is this necessary?
If you're a programmer at Apple or a contributer to one of the biggest open source languages, hopefully you've taken any math or computer science class and can understand what _iff_ means.

#gardenerout